examples/realitytabs.html

```diff
@@ -1767,6 +1775,7 @@
                         const pressed = gamepad.buttons[1].pressed;
                         const lastPressed = lastPresseds[i];
                         if (pressed && !lastPressed) {
+                          moveMode = true;
```

Comment:
Move mode is already tracked in `moves[i]`, per-controller, so why nod just use that [here](https://github.com/webmixedreality/exokit/pull/714/files#diff-bdc418c11110bb6bec917c75b88495f9R1898)?

Reply:
fixed 
f3848f1